KillTeam/Models/Member.cs

```diff
@@ -533,7 +533,7 @@ public static async Task<Member> DuplicateFrom(string memberId, string teamId =
                     int i = 1;
                     while (equipe.Members.Where(m => m.Name == membreNom).Count() == 1)
                     {
-                        membreNom = membre.Name + " - " + Resx.Translate.Copier + " (" + i + ")";
+                        membreNom = membre.Name + " - " + Properties.Resources.Copier + " (" + i + ")";
                         i++;
                     }
                 }

```

Comment:
We might want to tidy up these properties and use expressions where we can i.e.

`[JsonIgnore]
public string Blessure => Resx.Translate.Blessures + " : ";`

Reply:
Yes we do.
That's the next step ;)